otx/algorithms/detection/configs/instance_segmentation/efficientnetb2b_maskrcnn/deployment.py

```diff
@@ -6,10 +6,10 @@
 
 ir_config = dict(
     output_names=["boxes", "labels", "masks"],
-    input_shape=(1024, 1024),
+    # input_shape=(1024, 1024),
```

Comment:
Why we comment this?

Reply:
Then, why don't we just remove this line, not commenting

Reply:
It could be deleted. I didn't comment on this line. So, I just leave this line. I'll delete this

Reply:
https://github.com/openvinotoolkit/training_extensions/pull/2172/commits/9694b18b3e292379f8d278d0a4cf517d26d562db